external/storm-mqtt/src/main/java/org/apache/storm/mqtt/common/MqttOptions.java

```diff
@@ -88,7 +87,7 @@ public String getWillTopic() {
      * If set the server will publish the client's Will message to the specified topics if the client has an unexpected
      * disconnection.
      *
-     * @param willTopic
+     * @param willTopic the will topic
```

Comment:
This is minor, but does removing the `@param` cause the checkstyle check to fail? If not, I'd rather get rid of these until someone has something useful to write here. "the will topic" is just noise.

Reply:
Good catch. I though that parameters would be enforced as soon as Javadoc is present. I agree 110% that those are pointless.